keymap.cson

```diff
@@ -26,6 +26,8 @@
 
 # window/pane navigation
 'body, atom-text-editor':
+  'shift-cmd-y': 'pane:show-next-item'
+  'shift-cmd-h': 'pane:show-previous-item'
```

Comment:
I'm a fan of keeping these standard `cmd-shift-[` style, they work in chrome and safari.